rero_ils/modules/local_fields/jsonschemas/local_fields/local_field-v0.0.1.json

```diff
@@ -0,0 +1,166 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Local fields",
+  "description": "Local fields",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "fields"
+  ],
+  "propertiesOrder": [
+    "fields"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate local_field records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/local_fields/local_field-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Local field ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "organisation": {
+      "title": "Organisation",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "Organisation",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/organisations/.*?$"
+        }
+      }
+    },
+    "parent_ref": {
+      "title": "Parent ref",
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "$ref"
+      ],
+      "properties": {
+        "$ref": {
+          "title": "Patron type URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/documents|items|holdings/.*?$"
+        }
+      }
+    },
+    "fields": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "field_1"
+      ],
+      "propertiesOrder": [
+        "field_1",
+        "field_2",
+        "field_3",
+        "field_4",
+        "field_5",
+        "field_6",
+        "field_7",
+        "field_8",
+        "field_9",
+        "field_10"
```

Comment:
Why just not an array of "field" with `minItems: 1` and `maxItems:10` ?
Can we place a data for "field_8" without filling the seventh first ?

Reply:
Yes